CppSamples/Layers/ConfigureClusters/ConfigureClusters.cpp

```diff
@@ -247,27 +245,36 @@ void ConfigureClusters::mouseClicked(QMouseEvent& mouseEvent)
     // clear cluster selection
     if (m_aggregateGeoElement)
       m_aggregateGeoElement->setSelected(false);
-
-    for (Popup* popup: result->popups())
+    if (!result->popups().empty())
```

Comment:
empty works, but `isEmpty` is more idiomatic with Qt containers. I would go with `isEmpty`.